apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -102,7 +102,10 @@ DropletBlockTooltipManager.prototype.installTooltipsForCurrentCategoryBlocks_ =
       }.bind(this)
     });
 
-    // Store the title/funcName as a block id so we can attach callouts later:
+    // Store the title/funcName as data-block so we can attach callouts later:
+    $(blockHoverDiv).attr('data-block', funcName);
```

Comment:
I had read earlier today that the `data` method would not persist it in the HTML (which I thought could be an issue for reading from outside of jquery), but now I'm wondering if that information is out of date.